File: Coding Club Beginner Projects/Lesson2/Guessing Game/guess.js
console.log("Guessing Game!\n\n");

var randomNum = Math.random() * 100;

var userGuess = prompt("Please enter in your guess: ") * 1;

if (userGuess === randomNum)
{
    console.log("You guessed correctly! Congratulations!")
}
else
{
    console.log("You guessed wrong...");
}

File: Coding Club Beginner Projects/Lesson2/Guessing Game/guess.py
import random

print("Guessing Game!\n")

user_guess = int(input('Please enter in what you think the number is: '))

random_num = random.randint(1, 100)

if user_guess == random_num:
    print("You guessed correctly! Congratulations!")
else:
    print("You guessed wrong...")